### PR TITLE
fletcher:0.4.4

### PR DESCRIPTION
**Pull request by Jollywatt**
I am submitting
- [ ] a new package
- [x] an update for a package

A featureful update to fletcher, which in particular adds a user-extendable arrowheads/marks system and the ability to easily enclose nodes in groups.

Changes since v0.4.3:

- Support fully customisable marks/arrowheads!
  - Added new mark styles and tweaked some existing defaults. **Note.** This may change the micro-typography of diagrams from previous versions.
- Add node groups via the `enclose` option of `node()`.
- Node labels can be aligned inside the node with `align()`.
- Node labels wrap naturally when label text is wider than the node. **Note:** This may change diagram layout from previous versions.
- Add a `layer` option to nodes and edges to control drawing order.
- Add node shapes: `ellipse`, `octagon`.

### CODE REVIEW

File: packages/preview/fletcher/0.4.4/docs/gallery/algebra-cube.typ [hunk 1]
```diff
@@ -0,0 +1,47 @@
+#import "@preview/fletcher:0.4.2" as fletcher: diagram, node, edge
```
Comment (elegaanz) [inline packages/preview/fletcher/0.4.4/docs/gallery/algebra-cube.typ:1]: There are a few imports in the examples that are still using some older versions.

**Review by elegaanz — COMMENTED** [review]
Thank you :) just a small remark about outdated imports in your examples, that you may want to fix. But it is not mandatory to get this merged, I just thought that maybe you would like to update them.

File: packages/preview/fletcher/0.4.4/docs/gallery/algebra-cube.typ (hunk 1 above)
Comment (Jollywatt) [reply packages/preview/fletcher/0.4.4/docs/gallery/algebra-cube.typ]: Thanks! I've just fixed these.

**Comment by elegaanz** [comment]
Thank you!

**Pull request merged**